openrtb2/brand_version.go

```diff
@@ -0,0 +1,37 @@
+package openrtb2
+
+import "encoding/json"
+
+// 3.2.30 Object: BrandVersion
+//
+// Further identification based on User-Agent Client Hints, the BrandVersion object is used to identify a device’s browser or similar software component, and the user agent’s execution platform or operating system.
+type BrandVersion struct {
```

Comment:
This is actually an adcom1 object.

TODO: move to adcom1, and import adcom1 in openrtb2